pkg/daemon/ceph/client/upgrade.go

```diff
@@ -35,13 +35,14 @@ const (
 
 // CephDaemonsVersions is a structure that can be used to parsed the output of the 'ceph versions' command
 type CephDaemonsVersions struct {
-	Mon       map[string]int `json:"mon,omitempty"`
-	Mgr       map[string]int `json:"mgr,omitempty"`
-	Osd       map[string]int `json:"osd,omitempty"`
-	Rgw       map[string]int `json:"rgw,omitempty"`
-	Mds       map[string]int `json:"mds,omitempty"`
-	RbdMirror map[string]int `json:"rbd-mirror,omitempty"`
-	Overall   map[string]int `json:"overall,omitempty"`
+	Mon          map[string]int `json:"mon,omitempty"`
```

Comment:
Actually why not simply re-use cephv1 types instead of a new struct? Just consume `CephDaemonsVersions` with the API types.

Reply:
removed.